src/hooks/useCustomLoaders.ts

```diff
@@ -0,0 +1,14 @@
+import { useMemo } from "react";
+
+import { customLoadersSelector } from "utils/selectors";
+
+import { useSelector } from "./useSelector";
+import { useFirestoreConnect } from "./useFirestoreConnect";
+
+export const useCustomLoaders = () => {
+  useFirestoreConnect("loaders");
+
+  const customLoaders = useSelector(customLoadersSelector);
+
+  return useMemo(() => customLoaders ?? [], [customLoaders]);
+};
```

Comment:
Instead of using `useFirestoreConnect`, we can probably just statically load this once.

Might make sense to do it with a React context or similar so we can do it 'as early in the app load' as possible.

Might also make sense to cache the data/urls in localstorage so we can fetch from there the next time (depending on how long it actually takes to load this data from firebase?)

Reply:
added in https://github.com/sparkletown/sparkle/pull/1240/commits/4de28517e28e767968972cacb6b47f35f60a1989

Reply:
uses localStorage to cache/restore this data in https://github.com/sparkletown/sparkle/pull/1240/commits/0de969b334cc80d6890c7a88c6e5448b7fe1c0fb